### PR TITLE
Update Ali data on speakers.yml

### PR DESCRIPTION
**Pull request by aliakbars**

### CODE REVIEW

File: _data/speakers.yml [hunk 1]
```diff
@@ -658,17 +658,20 @@ In my spare time I like to go mountain biking, bouldering, do some woodworking o
...
   topic: My Journey in Learning and Relearning Bayesian Statistics
-  proposal_summary: My journey in learning (and relearning) Bayesian methods as a computer scientist
-  discourse_link:
-  youtube_link:
-  description: A data scientist and a lecturer.
-    Learning/teaching data science, machine learning, and artificial intelligence.
+  proposal_summary: In this talk, I will share my journey in learning (and relearning) Bayesian methods as a computer scientist
+    and the hurdles I've found along the way. I will also share some of the resources that I've found useful in my journey.
+  discourse_link:
+  youtube_link: https://www.youtube.com/channel/UCJvg7v0hwrXwaZP17Gh_O5w/
```
Comment (OriolAbril) [inline _data/speakers.yml:670]: This field is not used yet, but unlike github/twitter/web links, it should be a link to the posted talk, similarly to how `discourse_link` should not be a link to your profile but to the post where the discussion about your talk is happening.

**Review by OriolAbril — COMMENTED** [review]
Thanks for the update!

**Comment by canyon289** [comment]
Ali,
Thanks for the PR. Since we need to run this conference with consistency we'll incorporate your changes into another PR. Looking forward to your talk!

**Comment by canyon289** [comment]
Hey Ali,
I looked across all our pieces and saw that we already used this info across other platforms. Well try and get this change in a coordinated fashion but given the proximity to the conference need to focus on some other tasks! We will keep you updated though as things change

**Comment by aliakbars** [comment]
Hey Ravin,

Thank you for looking into this. That is totally understandable. Sorry for
the late updates.

Looking forward to hearing from you.

Best,
Ali

**Comment by canyon289** [comment]
Ali,
No need to be sorry! We'll try and work these updates in. Just give us a bit to get conference kickstarted and I'll come back to this.

**Pull request closed without merging**